### ISSUE TITLE
a simple working example

### ISSUE THREAD

**Issue by thewayur** [pixijs/pixi-layers#66]
i m new to canvas, pixi, gsap, pixi-particles and now this pixi-layers. so pardon me for this noob que,

i tried to open light example, but it seems to be outdated.

so can u guyz please give me a very simple example in a single page with pixi 6, gsap 3 and ur current pixi layers?

what i m asking to sort layers on demand at run time.
thanks

**Comment by ivanpopelyshev** [comment]
I'll look into this today or tomorrow :)

**Comment by ivanpopelyshev** [comment]
Wait, do you mean lights with normals?

https://pixijs.io/examples/#/plugin-layers/lighting.js works fine

**Comment by thewayur** [comment]
> Wait, do you mean lights with normals? we dont have it for pixi-v5 and v6 :(
> 
> Everything else works - https://pixijs.io/examples/#/plugin-layers/lighting.js works fine

yes , this example working now, i m silly (my computer as well i.e. loading from cache)
well this example working fine with my localhost. THANKS FOR EXAMPLE

but again, i m so confused because i m not able to figure out how to rearrange individual bunny. sorry i m new in pixi

**Comment by thewayur** [comment]
> Wait, do you mean lights with normals? we dont have it for pixi-v5 and v6 :(
> 
> Everything else works - https://pixijs.io/examples/#/plugin-layers/lighting.js works fine

sorry for m mistake, actually i had hard reloaded 5 6 times and the assests were loading from cache, hence not rendering any bunny.. weird chrome cache
